client/src/theme/styles/components/reader/notes/filteredList.js

```diff
@@ -213,11 +221,15 @@ export default `
 
   &__item-tag {
     margin-top: 12px;
+    width: 100%;
 
     ${respond(
       `margin-top: -3px;
-    margin-left: 40px;`,
-      95
+    margin-left: 24px;
+    width: auto;
+    flex-basis: ${TAG_WIDTH};
+    `,
+      120
```

Comment:
See my note in linear re: refactoring this to a table instead of adding this flex-basis.